src/proxy/BaseProxy.ts

```diff
@@ -26,7 +26,7 @@ import ClientMessage = require('../ClientMessage');
  */
 export class BaseProxy {
 
-    protected client: HazelcastClient;
+    public client: HazelcastClient;
```

Comment:
Why do we need to access client out of a proxy object publicly?

Reply:
To access in `ReliableTopicListenerRunner`:
` this.logger = this.proxy.client.getLoggingService().getLogger();`